src/templates/editor_inactivity.html

```diff
@@ -77,17 +77,26 @@
               </p>
               <p class="header-message">
                 We haven't seen any activity from your account this past week.
-                Editors are expected to contribute at least one paper and comment per week.
+                Editors are expected to contribute at least three meaningful contributions per week.
```

Comment:
What are 3 meaningful contributions? 

What are the exact requirements here?

Reply:
@lightninglu10 
In the slack - [Meaningful contributions include:](https://researchhub-community.slack.com/archives/G01DP8EMCGN/p1646078198722159?thread_ts=1646056936.540369&cid=G01DP8EMCGN)
```
(1) upload (paper) + summary (comment),or a
(2) response to another user (comment)
```

Reply:
I think we should define the meaningful contributions in this email

Reply:
@lightninglu10 Updated 
<img width="962" alt="image" src="https://user-images.githubusercontent.com/364060/157951584-d1a82143-21ad-4f87-a5c8-5a6c2145d93d.png">

Reply:
the visit researchhub button looks weird -- can you make the text always white?

Reply:
Could we also get a <br /> element in between Your Contributions: and the next line, same with Meaningful Contributions include:

Just to add another line of space between them?

Reply:
@lightninglu10 Updated
<img width="934" alt="image" src="https://user-images.githubusercontent.com/364060/157966861-0ac79556-9dd7-40f4-b9a7-32b7de5bf1fe.png">